.gitattributes

```diff
@@ -1,9 +1,10 @@
-/.github/               export-ignore
-/.phive/                export-ignore
-/tests/                 export-ignore
-/.editorconfig          export-ignore
-/.gitattributes         export-ignore
-/.gitignore             export-ignore
-/.php-cs-fixer.dist.php export-ignore
-/box.json.dist          export-ignore
-/phpunit.xml.dist       export-ignore
+/.github/                       export-ignore
+/.phive/                        export-ignore
+/tests/                         export-ignore
+/.editorconfig                  export-ignore
+/.gitattributes                 export-ignore
+/.gitignore                     export-ignore
+/.php-cs-fixer.dist.php         export-ignore
+/box.json.dist                  export-ignore
```

Comment:
maybe not configure indent to not have the wall file change ?

Reply:
Not a big problem, this file is not going to change a lot, @Ciloe.